README.md

```diff
@@ -29,6 +30,42 @@ $ yarn add @shopify/preact-testing
 
 This library allows you to test Preact components with a focus on type safety and testing based on a component’s external API. In order to keep the API small and easy-to-use, it will generally track to only the latest minor release of Preact.
 
+### Basic
+
+A testcase using `@shopify/preact-testing` tends to have more or less the following structure:
+- [`mount`](#mount) your component with some props to get a wrapper
+- (optionally) mutate it using the wrapper's [`setProps`](#setProps) or by [`trigger`](#trigger)ing callbacks on elements within it
+- make assertions based on the wrapper (often using the custom [matchers](#matchers))
+
+#### Example
+
+This example uses [jest](https://jestjs.io/) as a test-runner.
+
+```tsx
+import {h} from 'preact';
```

Comment:
delete since its not being use?

Reply:
This is the equivalent of needing `import React from 'react'` for JSX in a React app, it is used because there is JSX in the file.

Reply:
I see, just looks a little funny with destruction. As long as linter wont complain about unused variable, its 👍

Reply:
it won't if you set it and typescript up right for preact